applications/Unified_Liquidity_Balance_API.md

```diff
@@ -0,0 +1,158 @@
+
+### Contact
+
+- **Contact Name:** Leonardo Razovic
+- **Contact Email:** leonardo@polimec.org
+- **Website:** [Polimec](https://app.polimec.org/)
+
+### Legal Structure
+
+- **Registered Address:** Reiffergässli 4, 6300 Zug, Switzerland
+- **Registered Legal Entity:** Polimec Foundation
+
+### Team’s Experience
+
+Our team has extensive experience in cloud‑native TypeScript and Rust development. We’ve contributed to Polkadot JS API, authored Polkadot SDK/FRAME pallets, and deployed production workloads on Cloudflare Workers.
+
+### Team Code Repo
+
+- https://github.com/Polimec/polimec-node
+- https://github.com/Polimec/ulb-api
+
+## Development Status :open_book:
+
+- Proof‑of‑concept implemented and deployed on Cloudflare Workers.
+- Codebase: https://github.com/Polimec/ulb-api
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):** 2
+- **Total Costs:** 27,000 USD
+- **DOT %:** 60%
+
+### Milestone 1 — Core SSE Service
+
+- **Estimated duration:** 1 month
+- **FTE:** 2
+- **Costs:** 9,000 USD
+
+|  Number | Deliverable                 | Specification                                        |
+| ------: | --------------------------- | ---------------------------------------------------- |
+| **0a.** | License                     | MIT                                                  |
+| **0b.** | Documentation               | Inline docs + Quickstart tutorial                    |
+| **0c.** | Testing and Testing Guide   | Unit tests + test runner guide                       |
+|      1. | SSE Server                  | Cloudflare Worker exposing GET /balances/{accountId} |
+|      2. | Durable Object Listener     | Manages subscriptions, reconnections                 |
+|      3. | Chain Adapters              | Polkadot & Asset Hub                                 |
+|      4. | Balance Aggregation Service | Merges per‑chain balances                            |
+|      5. | JS/TS Client Example        | Browser usage sample                                 |
+
+### Milestone 2 — Multi‑Chain Support & Advanced Features
+
+- **Estimated duration:** 1 month
+- **FTE:** 2
+- **Costs:** 9,000 USD
+
+|  Number | Deliverable               | Specification                    |
+| ------: | ------------------------- | -------------------------------- |
+| **0a.** | License                   | MIT                              |
+| **0b.** | Documentation             | API reference + advanced guide   |
+| **0c.** | Testing and Testing Guide | Integration/stress tests + guide |
+|      1. | Additional Chain Adapters | Polimec & Hydration              |
+|      2. | Custom Filters            | Client‑specified chains/tokens   |
+|      3. | Heartbeat & Reconnection  | Enhanced reliability             |
+|      4. | Performance Optimization  | Latency & throughput benchmarks  |
+
+### Milestone 3 — Documentation, Tooling & Support
+
+- **Estimated duration:** 1 month
```

Comment:
```suggestion
- **Estimated duration:** 6 months
```

Reply:
Since the timeline for maintenance SLA would be 6 months, can we adjust total milestone duration to 6 months?